test/packet/scripts/add_vagrant_box.sh

```diff
@@ -1,55 +1,169 @@
 #!/bin/bash
 
-if [[ $# -ne 1 ]]; then
-    echo "usage: add-vagrant-boxes.sh [vagrant_box_defaults.rb path]"
-    exit 1
+set -e
+
+check_cmd() {
+    for cmd in $@ ; do
+        if !(command -v $cmd >/dev/null) ; then
+            echo "Error: $cmd not found."
+            exit 1
+        fi
+    done
+}
+check_cmd curl vagrant
+
+usage() {
+    echo -e "usage: add-vagrant-boxes.sh [options] [vagrant_box_defaults.rb path]"
+    echo -e "\tpath to vagrant_box_defaults.rb defaults to ./vagrant_box_defaults.rb"
+    echo -e ""
+    echo -e "options:"
+    echo -e "\t-a\t\tuse aria2c instead of curl"
+    echo -e "\t-b <box>\tdownload selected box (defaults: ubuntu ubuntu-next)"
```

Comment:
Maybe it should default to all, so `ubuntu-dev`, `ubuntu-next`, and `ubuntu` if I recall correctly.

Reply:
Ah, I guess you defaulted it this way to avoid changing the current behavior?